frontend/src/components/common/NullScreen.js

```diff
@@ -120,6 +121,12 @@ const NullScreen = (props) => (
         <p>Create a contact using the menu above</p>
       </div>
     )}
+    {props.type === "incidents" && (
```

Comment:
You should add to and use NullScreen component instead of having No incidents found here.

Reply:
This is the nullscreen component, but right now we pass the nullscreen the thing it is showing null for... I think this nullscreen should be looked, because this pattern is not sustainable, nor that helpful.

Reply:
lol derp